task_runner

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+TARGET=skbio
+
+if [ ${2} ]; then
+  TARGET=${2}
+fi
+
+COMMAND=${1}
+
+case "${COMMAND}" in 
+  "tests" )
+    set -x
+    nosetests ${TARGET} --with-doctest --ignore DO_NOT_IGNORE_ANYTHING
+    pep8 ${TARGET}
+    flake8 ${TARGET}
+    ./checklist.py
+    ;;
+  "docs" )
+    set -x
+    pushd doc
+    make html
```

Comment:
Once in a while, we need to generate the sphinx source rsts from scratch. Would you mind adding something like a "docs --clean" which would run `make clean` before `make html`?